docs/source/api/aihwkit.inference.converter.rst

```diff
@@ -14,3 +14,4 @@ Submodules
 
    aihwkit.inference.converter.base
    aihwkit.inference.converter.conductance
+   aihwkit.inference.converter.fusion
```

Comment:
This change (and the others in this folder) aim to tackle the warnings similar to:
```
/home/docs/checkouts/readthedocs.org/user_builds/aihwkit/checkouts/latest/docs/source/api/aihwkit.inference.converter.fusion.rst: WARNING: document isn't included in any toctree
```